src/libs/validationHandler.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response, NextFunction } from 'express';
+export default (config) => (req: Request,res: Response,next: NextFunction) => {
+    const errors = [];
+    Object.keys(config).forEach((key) => {
+        const i = 0;
+        const keys = config[key];
+        const locations = keys.in[i];
+        let request = req[locations][key];
+        const regex = keys.regex;
+        if ((keys.required) && !(request)) {
+            const err = {
+                key: `${key}`,
+                location: `${keys.in}`,
+                errorMessage: `${keys.errorMessage || 'required'}`
+                };
+            errors.push(err);
+        }
+        if ((!keys.required) && !(request)) {
+            return request = keys.default;
+        }
+        if (
+            (((keys.number) && !(Number.isInteger(Number(request)))) ||
```

Comment:
Number.isInteger will only allow integer value use isNaN() instead it will allow both integer and float values.